### DISCUSSION TITLE
Webhook Component Docs

### DISCUSSION THREAD

**Discussion by Mareddie** [symfony/symfony#50825] [Q&A] [answered]
Hi,

I have a question regarding the new Webhook and RemoteEvent components. I've recently upgraded to new Symfony 6.3 and I saw on the blog that this new version is coming out with these new components that I'd love to implement in my codebase.

However, in [Webhook component repo](https://github.com/symfony/webhook), there's still a note that this component is experimental. Is that correct?

Also, when I search the Symfony documentation (or even google it), there is no documentation for this new component. When will there be some?

Thanks.

**Comment by Mareddie** [comment]
As of now, the link for [documentation in component repository](https://symfony.com/doc/current/webhook.html) leads to 404 page.

**Comment by lyrixx** [comment, marked as answer]
Hello,

Yes, the component is still experimental 

And the doc is not merged yet https://github.com/symfony/symfony-docs/pull/17964

**Reply by Mareddie** [reply]
Thanks!
